### PR TITLE
Changed to use HTTPS

### PR DESCRIPTION
**Pull request by PhDuck**
There are a bunch of minor changes, but these just remove lines with trailing spaces.

The real meat of this pull request is the changes made to calls, instead of using HTTP calls we now use HTTPS, this is relevant for users that use https://imgur.com/ if you are using HTTPS, and the extension then makes HTTP calls it will fail, since you can't just cross into unsafe connections.

### CODE REVIEW

**Comment by PhDuck** [comment]
For example all users using the https://www.eff.org/Https-everywhere plugin should have problems.

File: main/icepage.js [hunk 1]
```diff
@@ -110,32 +110,32 @@ function favoritesPage(){
 				}
 				var allListener = makeListItem(true, "all");
 				allListener.click(function() {
-					window.location.href = "http://imgur.com/account/favorites";
```
Comment (PhDuck) [inline main/icepage.js:113]: Change to HTTPS

File: main/icepage.js [hunk 2]
```diff
@@ -247,7 +247,7 @@ function galleryPage() {
 
 function getAllFavePosts(callback) {
 	function getPostsFromPage(pageNum, allFaves) {
-		$.get("http://imgur.com/account/favorites/newest/page/" + pageNum + "/miss?scrolled", function(htmlz) {
```
Comment (PhDuck) [inline main/icepage.js:250]: Change to HTTPS

**Comment by GrimTheReaper** [comment]
As much as I want to push for security, shouldn't it check what the current connection to imgur is and use that for TLS options? Some places might block 443, or HTTPS.

**Comment by anotherglitchinthematrix** [comment]
@GrimTheReaper we can use the protocol-relative url like "//www.imgur.com", there is some issues as I know but this is a chrome extension thus I'm sure there will be no problem.

**Comment by PhDuck** [comment]
Sounds like a solid idea @alicanyildiz I will commit and push when I get off work.

**Comment by GrimTheReaper** [comment]
Connects to https://github.com/kyrofox/iceproject/issues/7

**Comment by shea-sollars** [comment]
use [protocol-relative URLs](http://www.paulirish.com/2010/the-protocol-relative-url/); just take out all `http:` and `https:`, leaving the `//url-path.com/png`.

**Comment by PhDuck** [comment]
@shea-sollars Yes, as previously mentioned by @alicanyildiz. I think the fastest would just be to close this, and create a new pull request where protocol-relative URLs are used.

**Pull request closed without merging**